### DISCUSSION TITLE
RealTraffic

### DISCUSSION THREAD

**Discussion by snodawg** [albar965/littlenavmap#883] [Ideas]
Have you ever considered tying in with The RealTraffic plugin that works with Live Traffic for X-plane & works with FS2O20. Real Traffic shows real time real world actual traffic in x-plane and on the RealTraffic built in map. Would be awesome to see the real time traffic in litte nav map. flyrealtraffic dot com

**Comment by albar965** [comment]
Does it fill the datarefs of the X-Plane [TCAS interface](https://developer.x-plane.com/article/overriding-tcas-and-providing-traffic-information/)? LNM cannot show anything if the plugin just injects 3D models and does not fill these datarefs.

**Reply by snodawg** [reply]
I'm not sure about the data refs ? But the Real Traffic shows up on the display as well out the window. I just thought it would be cool to have real traffic show up in Little Nav Map as well if possible. Here's his web site and and some dev info. https://www.flyrealtraffic.com/RTdev2.0.pdf


Sorry if its not something you would be interested in, just thought it might be an awesome addition. Disregard if I wasted any of your time. Not my intention.

**Reply by albar965** [reply]
It's up to the Live Traffic developers to provide information in the X-Plane standard interfaces. Aircraft should show up in LNM if they do.

Using Real Traffic without the Live Traffic plugin? Not sure if this makes sense to see traffic in LNM without having it in the simulator.
